workspace/utilities/lib/picture.xsl

```diff
@@ -3,93 +3,26 @@
-
 	<xsl:template name="picture">
 		<xsl:param name="image" select="image" />
 		<xsl:param name="alt" select="alt" />
-		<xsl:param name="sizes" select="exsl:node-set($default-image-sizes)/sizes" />
-		<xsl:param name="use-sizes" select="$image/@type != 'image/gif' and $image/@type != 'image/svg+xml' and $image/@type != 'image/svg' and $image/@type != 'image/tiff' and $image/@type != 'text/plain'"/>
 		<xsl:param name="ext-attr" />
 		<xsl:param name="ext-attr-image" />
 
 		<xsl:variable name="attr">
 			<xsl:copy-of select="$ext-attr" />
-			<add dev-component="picture" />
-		</xsl:variable>
-
-		<xsl:variable name="attr-img">
-			<add class="block width-full" />
-			<set src="/workspace{$image/@path}/{$image/filename}" />
-			<set alt="{$alt}" />
-			<set loading="lazy" />
 			<xsl:copy-of select="$ext-attr-image" />
-			<add dev-element="img" />
 		</xsl:variable>
 
-		<xsl:call-template name="element">
-			<xsl:with-param name="attr" select="$attr" />
-			<xsl:with-param name="element" select="'picture'" />
-			<xsl:with-param name="content">
-
-				<xsl:if test="$use-sizes">
-					<xsl:apply-templates select="$sizes/size" mode="picture-source">
-						<xsl:with-param name="image" select="$image" />
-					</xsl:apply-templates>
-				</xsl:if>
-
-				<xsl:call-template name="element">
-					<xsl:with-param name="attr" select="$attr-img" />
-					<xsl:with-param name="element" select="'img'" />
-				</xsl:call-template>
-			</xsl:with-param>
+		<xsl:call-template name="image">
+			<xsl:with-param name="image" select="$image" />
+			<xsl:with-param name="alt" select="$alt" />
+			<xsl:with-param name="ext-attr" select="$attr" />
 		</xsl:call-template>
-	</xsl:template>
-
-	<xsl:template match="size" mode="picture-source">
-		<xsl:param name="size" select="." />
-		<xsl:param name="image" />
-		<xsl:param name="srcset">
-			<xsl:call-template name="picture-src">
-				<xsl:with-param name="size" select="$size" />
-				<xsl:with-param name="image" select="$image" />
-			</xsl:call-template>
-		</xsl:param>
 
-		<xsl:variable name="attr">
-			<set srcset="{$srcset}" />
-			<set type="{$image/@type}" />
-			<set media="{$size/@media}" />
-			<add dev-element="picture-source" />
-		</xsl:variable>
-
-		<xsl:call-template name="element">
-			<xsl:with-param name="attr" select="$attr" />
-			<xsl:with-param name="element" select="'source'" />
-		</xsl:call-template>
-	</xsl:template> <!-- picture-source -->
-
-	<xsl:template name="picture-src">
-		<xsl:param name="image" />
-		<xsl:param name="size" />
-		<xsl:text>/image/1/</xsl:text>
-		<xsl:value-of select="$size/@request-height" />
-		<xsl:text>/</xsl:text>
-		<xsl:value-of select="$size/@request-width" />
-		<xsl:value-of select="$image/@path" />
-		<xsl:text>/</xsl:text>
-		<xsl:value-of select="$image/filename" />
-	</xsl:template> <!-- picture-src -->
+		<xsl:if test="$debug">
+			<pre>The XSLT template picture has been deprecated. Please use the XSLT template image.</pre>
+		</xsl:if>
```

Comment:
Use xslt message ?

Reply:
nope. https://developer.mozilla.org/en-US/docs/Web/XSLT/Element/message

I just want a simple dom element. It's more visual.

Reply:
How cant it be more visual then a fatal error ?

Reply:
my objective was to make this new template retro-compatible with picture. That way we can still output images and show a simple message if we are in debug mode.